osquery/tables/system/windows/video_info.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <boost/algorithm/string.hpp>
+
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genVideoInfo(QueryContext& context) {
+  Row r;
+  QueryData results;
+
+  WmiRequest wmiSystemReq("SELECT * FROM Win32_VideoController");
+  std::vector<WmiResultItem>& wmiResults = wmiSystemReq.results();
+  if (!wmiResults.empty()) {
```

Comment:
I'd say similar to our discussions on other tables, you can make this table a little tighter by flipping your logic here and returning early. Additionally a helpful `LOG(WARNING)` might be nice to explain the absence of data, and then you don't need to initialize all of the `"-1"`.
```
if (wmiResults.empty()) {
  LOG(WARNING) << "Failed to lookup video card information";
  return {};
}
...
```

Reply:
Makes sense, forgot to update this table from when I wrote it. Done now